cmd/kubectl-testkube/commands/root.go

```diff
@@ -62,6 +63,20 @@ var RootCmd = &cobra.Command{
 	Short: "Testkube entrypoint for kubectl plugin",
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		ui.SetVerbose(verbose)
+
+		cfg, err := config.Load()
+		ui.ExitOnError("loading config", err)
```

Comment:
will it run without kubernetes cluster and ability to load/store testkube config?